src/platform/qpg/CHIPDevicePlatformConfig.h

```diff
@@ -30,6 +30,10 @@
 
 #if CHIP_ENABLE_OPENTHREAD
 #define CHIP_DEVICE_CONFIG_ENABLE_THREAD 1
+#define CHIP_DEVICE_CONFIG_ENABLE_EXTENDED_DISCOVERY 1
+#define CHIP_DEVICE_CONFIG_ENABLE_COMMISSIONABLE_DISCOVERY 1
```

Comment:
Why would these be conditional on openthread?

Reply:
Copied from the EFR32 platform changed now